.eslintrc.js

```diff
@@ -1,35 +1,19 @@
 module.exports = {
-    extends: ['eslint:recommended', 'airbnb-base'],
```

Comment:
We could use the airbnb typescript rules set but I am slightly in favour of the more official rules.

Reply:
Yes, completely agree 👍